### ISSUE TITLE
[a11y]: Violation Tab scroll right and scroll right and left raise

### ISSUE THREAD

**Issue by lee-chase** [carbon-design-system/carbon#11109]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

7.56.0

### React version

16

### Automated testing tool and ruleset

 IBM Equal Access Accessibility Checker (Chrome/Firefox extension) with rule set "Latest Deployment" / accessibility guidelines "IBM Accessibility"

### Description

Aria hidden wrapping button raising accessibility violation `Element "button" should not be focusable within the subtree of an element with an 'aria-hidden' attribute with value 'true'`.

<img width="824" alt="image" src="https://user-images.githubusercontent.com/15086604/160808340-4f4b8d75-fac0-4469-b5bf-04fc8c2f8bad.png">

### WCAG 2.1 Violation

1.3.1 and 4.1.2

### CodeSandbox example

https://codesandbox.io/s/gallant-gauss-v5ntb0?file=/src/index.js

### Steps to reproduce

1. Open the code sandbox.
2. View the output in a window by itself. https://v5ntb0.csb.app/
3. Shrink the window until the scroll right or left is visible.
4. Run the accessibility scan and the results currently show the following

<img width="674" alt="image" src="https://user-images.githubusercontent.com/15086604/160810157-9c66df90-d305-432d-8b98-a16245d62a65.png">

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by abbeyhrt** [comment]
Hey @lee-chase! 👋  This rule has been removed from the most recent ruleset from IBMa, I think they found it to be too general or it led to too many false positives so this violation should be resolved! I ran the accessibility checker with the latest ruleset against your sandbox and the violation is no longer present. Going to close this but let me know if you are still running into this violation and we can keep investigating!
